Contributing.md

```diff
@@ -82,4 +82,38 @@ to the TC and the TC uses its standard consensus seeking process to evaluate whe
 not to add this new member. Members who do not participate consistently at the level of
 a majority of the other members are expected to resign.
 
-
+# Project Captains
+
+The Express TC can designate captains for individual projects/repos in the
+organizations. These captains are responsible for being the primary
+day-to-day maintainers of the repo on a technical and community front.
+Repo captains are empowered with repo ownership and module publication rights.
+When there are conflicts, especially on topics that effect the Express project
+at large, captains are responsible to bring it up with the TC and drive
+those conflicts to resolution. Captains are also responsible in making sure
+community members follow the community guidelines, maintaining the repo
+and the registry, as well as in providing user support.
+
+To become a captain for a project you will be expected to participate in that
+project for at least 6 months prior to the request. You should have helped with
```

Comment:
Should this be "for at least 6 months in the role of committer"?

Reply:
we did not specifically call this out in the TC. Should we? because there is an approval process, adding such an additional criteria may be seen as redundant?

Reply:
My comment is because I assume that in order to actually function as a repo owner one would need to be a committer? Because if a user had never been a committer before and suddenly was a repo owner, that would seem odd, as one would think that they should at least have non-zero experience landing commits into the repo?

Reply:
Anyway, the main reasons why it jumped out to me is that the TC is a progression of committer -> TC member, so thought that maybe that would make sense for Repo Captain as well. It just felt natural for some reason, which is the only reason I asked. It also felt like it may help better clarify what "participate" meant if that came up in the future as a point of content (did X actually "participate" for 6 months, etc.). I just felt like if the pre-req felt less fuzzy, a user would be more confident requesting the role by better knowing if they met it or not, idk, haha

Reply:
So are there no comments on this still? If not, I think the wording should be clarified in this manor. For example, in the TC section, it says that committers elect other committers to the TC. If we do not spell out somewhere that a repo captain is also a committer, then it is not clear that a repo captain can be a TC member unless they also separately become a committer.

I think we should clarify that repo captains are a subset of committers, just like TC members are.